src/gino/__init__.py

```diff
@@ -14,18 +16,14 @@
 def create_engine(*args, **kwargs):
     """
     Shortcut for :func:`sqlalchemy.create_engine` with ``strategy="gino"``.
-
     .. versionchanged:: 1.1
        Added the ``bakery`` keyword argument, please see :class:`~.bakery.Bakery`.
-
     .. versionchanged:: 1.1
        Added the ``prebake`` keyword argument to choose when to create the prepared
        statements for the queries in the bakery:
-
```

Comment:
I think this new line was to keep the markdown docs structure, the same for line 28.

Reply:
They had been removed by the formatter in my IDE. Has been fixed